Add a useful readme (#14)
adfinis-sygroup/ember-simple-auth-oidc: README.md

export default Route.extend(OIDCApplicationRouteMixin, {});
```

The `oidc-authentication-route-mixin` must cover the login / authentication route
(for example the Ember Simple Auth default `/login`).

```
// app/routes/login.js
import Route from "@ember/routing/route";
import OIDCAuthenticationRouteMixin from "ember-simple-auth-oidc/mixins/oidc-authentication-route-mixin";

export default Route.extend(OIDCAuthenticationRouteMixin, {});
```

The `oidc-end-session-route-mixin` must cover the logout / end session route
(for example `/logout`).

```
// app/routes/logout.js
import Route from "@ember/routing/route";
import OIDCEndSessionRouteMixin from "ember-simple-auth-oidc/mixins/oidc-end-session-route-mixin";

export default Route.extend(OIDCEndSessionRouteMixin, {});
```

To include authorization info in all Ember Data requests add the `oidc-adapter-mixin`
into the application adapter.

```
// app/adapters/application.js
import DS from "ember-data";
import OIDCAdapterMixin from "ember-simple-auth-oidc/mixins/oidc-adapter-mixin";

export default DS.JSONAPIAdapter.extend(OIDCAdapterMixin, {});
```

## Configuration

> The addon can be configured in the ember config with the key `ember-simple-auth-oidc`.

Reviewer: ```suggestion
The addon can be configured in your `environment.js` file with the key `ember-simple-auth-oidc`.
```

Author: How about `in the projects environment.js file`? Because I never used you/your until now.

Reviewer: Sounds good! Nitpick: `in the project's`

Author: ok, done

> At least the following config options need to be added to the ember config:

Author: Sounds better, changed this one as suggested

```
module.exports = function(environment) {
  let ENV = {
    modulePrefix: "oidc",
    environment,
    rootURL: "/",
    "ember-simple-auth-oidc": {
      host: "http://authorization.server/openid",
      clientId: "test",
      authEndpoint: "/authorize",
      tokenEndpoint: "/token",
      userinfoEndpoint: "/userinfo",
    }
  }
  return ENV;

> }

Reviewer: I'd focus on only the relevant section, maybe like this:
```js
module.exports = function(environment) {
  let ENV = {
    // ...
    "ember-simple-auth-oidc": {
      host: "http://authorization.server/openid",
      clientId: "test",
      authEndpoint: "/authorize",
      tokenEndpoint: "/token",
      userinfoEndpoint: "/userinfo",
    }
    // ...
  }
}
```

Author: Agree, also changed this one as suggested.

```

> A complete list of all possible config options can be found below:

Reviewer: I'd write it a bit more casually, maybe
```suggestion
Here's a complete list of all possible config options:
```

Author: Liked this one too, done
